docs/manual/hpx_main_usage.qbk

```diff
@@ -0,0 +1,55 @@
+[/=============================================================================
```

Comment:
This file needs to be added to the docs somehow, please see here for examples on how to do this: https://github.com/STEllAR-GROUP/hpx/blob/master/docs/hpx.qbk#L333-L359. Have you been able to build the docs?